src/apic-extension/azext_apic_extension/aaz/latest/apic/integration/_create.py

```diff
@@ -108,6 +122,27 @@ def _build_arguments_schema(cls, *args, **kwargs):
             help="The target lifecycle stage.",
             enum={"deprecated": "deprecated", "design": "design", "development": "development", "preview": "preview", "production": "production", "retired": "retired", "testing": "testing"},
         )
+
+        amazon_api_gateway_source = cls._args_schema.amazon_api_gateway_source
+        amazon_api_gateway_source.access_key = AAZStrArg(
+            options=["access-key"],
```

Comment:
`access-key` may cause confusions, maybe call it `amazon-access-key-key-vault-reference`? Please also consider give more clear name for the `secret-access-key` and `region-name` parameters

Reply:
parameters here won't exposed to users in helper, user can only use it if they use command as `--amazon-api-gateway-source access-key=XXX secret-access-key=XXX region-name=XXX`. This part is auto generated by aaz. Should we remove them?